softaware.Authentication.Hmac.AspNetCore/AuthenticationSchemeOptions.cs

```diff
@@ -11,6 +11,8 @@ public class HmacAuthenticationSchemeOptions : AuthenticationSchemeOptions
 
         public string AuthenticationScheme { get; set; }
 
+        public bool TrustProxy { get; set; }
```

Comment:
A short XML summary would be nice, something like

```csharp
/// <summary>
/// If <see langword="true"/>, the request scheme from the 'X-Forwarded-Proto' header is used to validate the request.
/// </summary>
```

Reply:
done